opencti-platform/opencti-front/src/utils/graph/EntityDetails.tsx

```diff
@@ -0,0 +1,301 @@
+            }
+            ... on Grouping {
+                name
+                description
+            }
+            ... on Individual {
+                name
+                description
+            }
+            ... on Organization {
+                name
+                description
+            }
+            ... on Sector {
+                name
+                description
+            }
+            ... on System {
+                name
+                description
+            }
+            ... on Indicator {
+                name
+                description
+            }
+            ... on Infrastructure {
+                name
+                description
+            }
+            ... on IntrusionSet {
+                name
+                first_seen
+                last_seen
+                description
+            }
+            ... on Position {
+                name
+                description
+            }
+            ... on City {
+                name
+                description
+            }
+            ... on AdministrativeArea {
+                name
+                description
+            }
+            ... on Country {
+                name
+                description
+            }
+            ... on Region {
+                name
+                description
+            }
+            ... on Malware {
+                name
+                first_seen
+                last_seen
+                description
+            }
+            ... on ThreatActor {
+                name
+                first_seen
+                last_seen
+                description
+            }
+            ... on Tool {
+                name
+            }
+            ... on Vulnerability {
+                name
+            }
+            ... on Incident {
+                name
+                first_seen
+                last_seen
+                description
+            }
+            ... on StixCyberObservable {
+                observable_value
+            }
+            ... on StixFile {
+                observableName: name
+            }
+            ... on Event {
+                name
+            }
+            ... on Case {
+                name
+            }
+            ... on Narrative {
+                name
+            }
+            ... on DataComponent {
+                name
+            }
+            ... on DataSource {
+                name
+            }
+            ... on Language {
+                name
+            }
+        }
+    }
+`;
+
+interface EntityDetailsComponentProps {
+  queryRef: PreloadedQuery<EntityDetailsQuery>
+}
+const EntityDetailsComponent: FunctionComponent<EntityDetailsComponentProps> = ({ queryRef }) => {
+  const classes = useStyles();
+  const { t } = useFormatter();
+
+  const entity = usePreloadedQuery<EntityDetailsQuery>(entityDetailsQuery, queryRef);
+  const { stixCoreObject } = entity;
+
+  return (
+    <div className={classes.entity}>
```

Comment:
It's maybe clearer to have directly { stixCoreObject && here to don't have all the if after
- { stixCoreObject?.entity_type}
- { stixCoreObject?.description
- etc

Reply:
Entities/relatiosn doesn't have same properties...so i wanted to avoid displaying empty properties...this is why there are condition on each property.

Reply:
I understand that, but to access to this property you need to have stixCoreObject != null

So you can have : 
if (!stixCoreObject) {
return <ErrorNotFound />
}
return (
...
{stixCoreObject.description
          && <div>
              <Typography
                variant="h3"
                gutterBottom={true}
                className={classes.label}
              >
                  {t('Description')}
              </Typography>
              <ExpandableMarkdown
                source={stixCoreObject?.description}
                limit={400}
              />
          </div>
...
)

Reply:
ok I just got it! fixed